src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -215,6 +222,9 @@ tag_invoke(boost::json::value_from_tag, boost::json::value& jv, LedgerEntryHandl
         {JS(index), output.index},
     };
 
+    if (output.deletedLedgerIndex) {
+        object["deleted_ledger_index"] = *(output.deletedLedgerIndex);
+    }
```

Comment:
I'd add a line of space below this block and perhaps remove the curlies for this one line `if` 👍